### PR TITLE
AC Automated Test - LRAC-9054 | master

### PR DESCRIPTION
**Pull request by MathausC**
https://issues.liferay.com/browse/LRAC-9054

### CODE REVIEW

**Comment by MathausC** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase [hunk 1]
```diff
@@ -127,12 +127,33 @@ definition {
...
+		}
 
+		task ("Click on user") {
+			ACSettings.clickUsersByEmail(userEmail = "${emailAdress}");
+
+			for (var email : list "${emailAdress}") {
+				ACUtils.searchBar(searchItem = "${email}");
+
+				AssertChecked.assertCheckedNotVisible(locator1 = "ACSettings#USER_LIST_CHECKBOX");
+			}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase:155]: Hi @MathausC , When you are going to do another PR, you could put in your test the macro to delete users that were created by Mateus [here](https://github.com/liferay-qa/liferay-portal/pull/943/commits/c750cbb297b35948ab45228e4d75f15863a5c32a)

**Comment by alexcandidos** [comment]
hi @ViniLopes20  , thanks for helping to review it.

Hi @MathausC , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward
